sql/2019/17_CDN/17_07.sql

```diff
@@ -0,0 +1,11 @@
+#standardSQL
+# 17_07: Count of sites with via header
+SELECT
+  _TABLE_SUFFIX AS client,
+  COUNT(respOtherHeaders) AS via
+FROM 
+  `httparchive.summary_requests.2019_07_01_*`
+WHERE
+   STRPOS(LOWER(respOtherHeaders), "via") > 0

```

Comment:
The `Via` header has its own field in the schema, `resp_via`, and IIRC enumerated fields are omitted from the `respOtherHeaders` field. So this condition could be simplified to something like `resp_via != ''`.

I would also remove this from the `WHERE` clause and use `COUNTIF` similar to the previous queries in the `SELECT` clause. This enables you to calculate the frequency, total, and percent values.

Reply:
Changed to resp_via != ' ' and COUNTIF